### PR TITLE
fix: bug linked to terraform version bump corrected

### LINKED ISSUE #202: Error when bumping terraform version to v1.6
There's a breaking change somewhere with terraform 1.6 : investigate, fix and update if possible.

```
│ Error: Retrieving AWS account details: AWS account ID not previously found and failed retrieving via all available methods. See https://www.terraform.io/docs/providers/aws/index.html#skip_requesting_account_id for workaround and implications. Errors: 2 errors occurred:
│       * retrieving caller identity from STS: operation error STS: GetCallerIdentity, https response error StatusCode: 0, RequestID: , request send failed, Post "https://sts.gra.amazonaws.com/": dial tcp: lookup sts.gra.amazonaws.com on 10.13.13.1:53: no such host
│       * retrieving account information via iam:ListRoles: operation error IAM: ListRoles, https response error StatusCode: 403, RequestID: a2d3bb0b-xxxxxxc2-4986-9ab6-e19be5fc82ac, api error InvalidClientTokenId: The security token included in the request is invalid.
```

### PR DESCRIPTION
**Pull request by heroinedor**
the AWS_ENDPOINT_URL_S3 should now contain the http/https prefix

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -157,8 +157,6 @@ jobs:
       # Install the latest version of Terraform CLI
       - name: Setup Terraform
         uses: hashicorp/setup-terraform@v2
-        with:
-          terraform_version: 1.4.7
```
Comment (ad2ien) [inline .github/workflows/deploy.yml:161]: I'd rather update the version here to avoid further unexpected inconsistencies between ovh and terraform

**Review by ad2ien — CHANGES_REQUESTED** [review]
Nice! Just a remark

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (heroinedor) [reply .github/workflows/deploy.yml:161]: version fixed to 1.6.X

**Review by ad2ien — APPROVED** [review]
:+1:

**Pull request merged**